src/Miscellaneous/Mis_GetTimeStep.cpp

```diff
@@ -205,6 +222,19 @@ double Mis_GetTimeStep( const int lv, const double dTime_SyncFaLv, const double
 #  endif
 
 
+// 1.9 CRITERION NINE : maximum velocity dS/dx ##ELBDM PHASE SOLVER ONLY##
+// =============================================================================================================
+#  if ( ELBDM_SCHEME == ELBDM_HYBRID )
+   if ( amr->use_wave_flag[lv] ) {
+      dTime[NdTime] = HUGE_NUMBER;
+   } else {
+      dTime[NdTime] = dTime_dt * ELBDM_GetTimeStep_Hybrid_Velocity( lv );
+   }
+
+   sprintf( dTime_Name[NdTime++], "%s", "Hybrid_Vel" );
+#  endif
```

Comment:
I think it's fine since `ELBDM_HYBRID` is also defined while `ELBDM_SCHEME` is only defined when `MODEL == ELBDM`. We have implicitly adopted similar assumptions in many other places.

If necessary, we can further do `undef ELBDM_SCHEME` at the end of `Macro.h`, similar to `LR_SCHEME`.

What do you think?

Reply:
That sounds promising. Let's undefined those variables.

Reply:
This kind of assumption is not specific to ELBDM. For example, I think we have also implicitly assumed that `UNSPLIT_GRAVITY` is defined only when enabling `GRAVITY`. So I prefer not to do that in this PR.

Reply:
Agree.